contribs/gmf/less/layertree.less

```diff
@@ -75,8 +51,15 @@
 
   .leaf,
   .group {
-    font-size: @font-size-small;
+    font-size: ceil(@font-size-base * 0.9);
```

Comment:
This is what bootstrap use (for font-size-small for example). I don't have any other good reason to do it.